test/distributed/test_dynamo_distributed.py

```diff
@@ -582,6 +584,45 @@ def opt_fn(inputs):
         opt_outputs = opt_fn(inputs)
         self.assertTrue(same(correct_outputs, opt_outputs))
 
+    @torch._inductor.config.patch({"layout_optimization": True, "keep_output_stride": False})
+    @patch.object(config, "optimize_ddp", True)
+    @unittest.skipIf(not has_triton(), "Inductor+gpu needs triton and recent GPU arch")
+    def test_graph_split_inductor_layout_optimizations(self):
+        assert config.optimize_ddp
+        channel_dim = 512
+        # channel dim must be > 64 for inductor to do layout optimization and use NHWC
+
+        class ToyModelConv(nn.Module):
```

Comment:
did you confirm that this model triggered the expected graph-splitting and exposed the strides bug?

Reply:
Yes, as per comment 
https://github.com/pytorch/pytorch/pull/114154#issuecomment-1819698382

When removing this assert https://github.com/pytorch/pytorch/pull/114154#issuecomment-1819673192, it fails with mismatched strides error